ci/verify.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+./vendor/bin/php-cs-fixer fix ./src
+./vendor/bin/psalm
+./vendor/bin/phpcs --config-set colors 1
```

Comment:
ok but did you try to remove this line and simply pass `--colors` to the command below?